api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestSingleShot.java

```diff
@@ -775,4 +775,44 @@ public void testNNFWTFLite() {
             fail();
         }
     }
+
+    @Test
+    public void testSNPE() {
+        if (!NNStreamer.isAvailable(NNStreamer.NNFWType.SNPE)) {
+            /* cannot run the test */
+            return;
+        }
+
+        try {
+            File[] model = APITestCommon.getSNPEModel();
+
+            TensorsInfo in = new TensorsInfo();
```

Comment:
Q: does snpe require in/out tensor information?

If this is mandatory, please consider to add 'snpe' https://github.com/nnstreamer/nnstreamer/blob/5e4406f36cd0a91b300f24d3aa9b7ee205ecaba0/api/capi/src/nnstreamer-capi-single.c#L622

Reply:
@jaeyun-jung No, it's not necessary. I changed the singleshot testcase to do not use the in/out tensor information. Please check